lib/judge/config.rb

```diff
@@ -8,24 +8,24 @@ class Config
     @@ignore_unsupported_validators = false
 
     def expose(klass, *attributes)
-      attrs = (@@exposed[klass] ||= [])
+      attrs = (@@exposed[klass.name] ||= [])
       attrs.concat(attributes).uniq!
     end
 
     def exposed
-      @@exposed
+      @@exposed.map { |k, v| [k.constantize, v] }.to_h
```

Comment:
Sounds better to change the test, yep. Aside: does this PR mean that either a constant _or_ a string can be used?